flopy/mf6/mfpackage.py

```diff
@@ -1347,7 +1385,10 @@ def set_all_data_internal(self, check_data=True):
             if (
                 isinstance(dataset, mfdataarray.MFArray)
                 or (
-                    isinstance(dataset, mfdatalist.MFList)
+                    (
+                        isinstance(dataset, mfdatalist.MFList)
+                        or isinstance(dataset, mfdataplist.MFPandasList)
```

Comment:
This could be simplified to `isinstance(dataset, (mfdataarray.MFArray, mfdatalist.MFList, mfdataplist.MFPandasList))`

Reply:
Simplified